troposphere/views/guacamole.py

```diff
@@ -0,0 +1,87 @@
+import json
```

Comment:
The Django/Python backend of Troposphere handles CAS and Globus authentication. We would have to have a CAS / Globus solution for nodejs/express to eliminate Django/Python.